code/modules/surgery/organs/subtypes/vox.dm

```diff
@@ -1,16 +1,37 @@
 /obj/item/organ/internal/liver/vox
+	name = "Vox liver"
 	alcohol_intensity = 1.6
 	species = "Vox"
 
 
 /obj/item/organ/internal/stack
 	name = "cortical stack"
-	icon_state = "brain-prosthetic"
+	icon_state = "cortical-stack"
 	parent_organ = "head"
 	organ_tag = "stack"
-	slot = "vox_stack"
+	slot =  "vox_stack"
 	robotic = 2
 	vital = 1
+	var/stackdamaged = 0
 
 /obj/item/organ/internal/stack/vox
-	name = "vox cortical stack"
\ No newline at end of file
+	name = "vox cortical stack"
+
+/obj/item/organ/internal/stack/vox/on_life()
+	if(damage < 1 && stackdamaged)
+		owner.mutations.Remove(SCRAMBLED)
+		owner.dna.SetSEState(SCRAMBLEBLOCK,0)
+		genemutcheck(owner,SCRAMBLEBLOCK,null,MUTCHK_FORCED)
+		stackdamaged = 0
+	..()
+
+
+/obj/item/organ/internal/stack/vox/emp_act(severity)
+	if(owner)
+		owner.mutations.Add(SCRAMBLED)
+		owner.dna.SetSEState(SCRAMBLEBLOCK,1,1)
+		genemutcheck(owner,SCRAMBLEBLOCK,null,MUTCHK_FORCED)
+		owner.AdjustConfused(20)
```

Comment:
40 seconds of confusion from one EMP is pretty brutal.

Reply:
I never get my time conversions right fox. I need to make me a chart.

Reply:
Spawn() and Sleep() use deciseconds (10 = 1 second), while statuses like confused are "cycles" (which average 2-4 seconds depending on what process they are in). The confusion is entirely understandable.

Reply:
Perhaps 4, then? We can guage what to set co fusion for here by looking where else in the code it pops up so we have examples.